library/card/card.css

```diff
@@ -23,13 +23,13 @@
     grid-area: cardActions;
 }
 
-.dui-card__info {
-}
+.dui-card__info {}
```

Comment:
can be removed if not in use.

Reply:
No, that's a placeholder for future use.